docs/css/tokens.css

```
@@ -4,11 +4,12 @@
   --ifm-navbar-padding-vertical: 0;
   --ifm-global-radius: 10px;
   --ifm-code-border-radius: 4px;
+  --ifm-code-padding-horizontal: 0.4em;
   --ifm-navbar-item-padding-vertical: 0;
   --ifm-font-family-base: "Geist", Inter, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.11.1)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)